packages/lib/models/Setting.ts

```diff
@@ -1121,7 +1121,24 @@ class Setting extends BaseModel {
 				storage: SettingStorage.File,
 				isGlobal: true,
 			},
-
+			imageResizing: {
```

Comment:
Do we need some description here?

Reply:
To be honest, here, I just copied and pasted some example and modify the values.
So, maybe there're some things to change.

Reply:
It looks good to me!